pkg/validator/config.go

```diff
@@ -215,6 +215,21 @@ func admitKDConfigCR(
 		)
 	}
 
+	// Populate the default ClusterSvcDomainBase if necessary
+	if configCR.Spec.ClusterSvcDomainBase == nil {
+		svcDomainBase := shared.DefaultSvcDomainBase
```

Comment:
Should this line be:

svcDomainBase := shared.GetDefaultSvcClusterDomainBase() in case it was set in the global config?

Reply:
This is the config processing here, so we're in the case where it was not set.

Reply:
Yep, you're right. Looked at it too quickly.